doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -0,0 +1,1408 @@
+    * The :c:func:`nrf_cloud_obj_location_request_create` and :c:func:`nrf_cloud_location_request` functions to accept the :c:struct:`nrf_cloud_location_config` structure in place of the ``bool request_loc`` parameter.
+
+* :ref:`lib_nrf_cloud_pgps` library:
+
+  * Fixed:
+
+    * A bug in prediction set update when the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_REPLACEMENT_THRESHOLD` Kconfig option was set to non-zero value.
+    * A bug in handling errors when downloading P-GPS data that prevented retries until after a reboot.
+    * A bug in handling errors when downloading P-GPS data does not begin due to active FOTA download.
+
+* :ref:`lib_nrf_provisioning` library:
+
+  * Added the :kconfig:option:`CONFIG_NRF_PROVISIONING_PRINT_ATTESTATION_TOKEN` option to enable printing the attestation token when the device is not yet claimed.
+  * Updated:
+
+    * Renamed nRF Device provisioning library to :ref:`lib_nrf_provisioning` library.
+    * The device mode callback sends an event when the provisioning state changes.
+    * The cbor code has been regenerated with zcbor 0.8.1 and adapted to API changes in zcbor 0.8.x.
+
+  * Fixed file descriptor handling by setting the :c:struct:`coap_client` structure's ``fd`` field to ``-1`` when closing the socket.
+
+* :ref:`lib_mqtt_helper` library:
+
+  * Added support for using a password when connecting to a broker.
+
+* :ref:`lib_lwm2m_client_utils` library:
+
+  * Updated:
+
+    * The Release Assistance Indication (RAI) support to follow socket state changes from LwM2M engine and modify RAI values based on the state.
+    * The library requests a periodic TAU (RPTAU) to match the client lifetime.
+
+* :ref:`lib_lwm2m_location_assistance` library:
+
+  * Updated the Ground Fix object to copy received coordinates to the LwM2M Location object.
+
+* :ref:`lib_fota_download` library:
+
+  * Added:
+
+    * The functions :c:func:`fota_download` and :c:func:`fota_download_any` that can accept a security tag list and security tag count as arguments instead of a single security tag.
+    * :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_CONNECT_FAILED` as a potential error cause in  :c:enumerator:`FOTA_DOWNLOAD_EVT_ERROR` events.
+
+* :ref:`lib_nrf_cloud_rest` library:
+
+  * Updated the :c:struct:`nrf_cloud_rest_location_request` structure to accept a pointer to a :c:struct:`nrf_cloud_location_config` structure in place of the single ``disable_response`` flag.
+
+* :ref:`lib_wifi_credentials` library:
+
+  * Updated PSA backend to use PSA Internal Trusted Storage (ITS) to store Wi-Fi credentials instead of Protected Storage.
+    The change is because PSA ITS is better suited to store secrets.
+    When switching storage, the credentials need to be migrated manually, or the existing credentials are lost.
+    Removed the ``CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET`` Kconfig option.
+    Use the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_OFFSET` Kconfig option instead.
```

Comment:
```suggestion
  * Updated the PSA backend to use the PSA Internal Trusted Storage (ITS) for storing of Wi-Fi credentials instead of the Protected Storage.
    This is changed because the PSA ITS is a better fit for storing assets like credentials.
    When switching storage, the credentials need to be migrated manually, or the existing credentials are lost.
  * Removed the ``CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET`` Kconfig option.
    Use the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_OFFSET` Kconfig option instead.
```

Reply:
Fixed.